triplestore-tdb-pool/src/main/java/zone/cogni/asquare/triplestore/pool/jenamemory/PollableLocalTdbRdfStoreService.java

```diff
@@ -0,0 +1,123 @@
+package zone.cogni.asquare.triplestore.pool.jenamemory;
+
+
+import org.apache.jena.atlas.RuntimeIOException;
+import org.apache.jena.query.TxnType;
+import org.apache.jena.tdb.StoreConnection;
+import org.apache.jena.tdb.base.file.FileException;
+import org.apache.jena.tdb.base.file.Location;
+import org.apache.jena.tdb.transaction.TDBTransactionException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import zone.cogni.asquare.triplestore.jenamemory.LocalTdbRdfStoreService;
+import zone.cogni.asquare.triplestore.pool.PoolableRdfStoreService;
+import zone.cogni.asquare.triplestore.pool.key.LocalTdbPoolKey;
+
+import java.io.File;
+import java.util.concurrent.TimeUnit;
+
+public class PollableLocalTdbRdfStoreService
```

Comment:
are we really having PollableLocalTdbRdfStoreService extend PoolableRdfStoreService? Seems like typo and if it is not it could be confusing?!

Reply:
My mistake, sorry, I couldn't see the `Poll` vs `Pool` typo :)